tests/e2e/tests/plugins/TypescriptPlugin.spec.ts

```diff
@@ -35,20 +34,19 @@ const topMenu: TopMenu = e2eContainer.get(CLASSES.TopMenu);
 const debugView: DebugView = e2eContainer.get(CLASSES.DebugView);
 const terminal: Terminal = e2eContainer.get(CLASSES.Terminal);
 const browserTabsUtil: BrowserTabsUtil = e2eContainer.get(CLASSES.BrowserTabsUtil);
-const workspaceNameHandler: WorkspaceNameHandler = e2eContainer.get(CLASSES.WorkspaceNameHandler);
 
-const devfileUrl: string = 'https://raw.githubusercontent.com/eclipse/che/main/tests/e2e/files/devfiles/plugins/TypescriptNodeDebug2PluginTest.yaml';
+const devfileUrl: string = 'https://github.com/SkorikSergey/web-nodejs-sample/tree/typescript-plugin';
```

Comment:
@SkorikSergey: what do you think about movement the file to eclipse/che repo?

Reply:
It is not just one file. We need whole repo for test. There are test files and extension.json file with enabled plugins. I planned  to discuss this with Balaji(how he plans to update all other plugins tests). It will not be so easy.

Reply:
@SkorikSergey: can we use https://github.com/che-samples/web-nodejs-sample/tree/devfilev2 instead of personal repo?

Reply:
Ok, I will move branches to che-samples and hope that no one will remove them.

Reply:
Thanks!

To make it safe, you could probably name branch like `devfile2-e2e-test`, and put some notes at the beginning of README.md .

Reply:
@dmytro-ndp Now tests use https://github.com/che-samples/web-nodejs-sample repo. I rechecked it on Eclipse Che 7.42.